portal-web/test/functional/com/liferay/portalweb/macros/LDAP.macro

```diff
@@ -127,6 +127,22 @@ definition {
 			userGroupName = "${userGroupName}");
 	}
 
+	@description = "Verify that the user is imported on localhost after LDAP sync, after a specified interval time"
+	macro verifyUserImportAfterImportIntervalTime {
+		if (!(isSet(intervalTime))) {
+			var intervalTime = "60000";
+		}
+
+        // Wait for the amount of the import interval time
```

Comment:
Ticket ID is missing from the text see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/poshi_pause_usage_check.markdown